purchase_supplierinfo_product_breakdown/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+* Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.
+* Enable **Use Product Components** option in the **Products** section of the **Purchase** tab.
+* In the **Purchase** tab of the **Product**  there will be a button added for the seller.
```

Comment:
If both product and vendor have the flag activated, in the **Purchase** tab of the **Product** > table "Vendor" there will be a button at the end of the supplierinfo line.